internal/distro/rhel9/images.go

```diff
@@ -361,9 +363,10 @@ func edgeSimplifiedInstallerImage(workload workload.Workload,
 	rawImg.Users = users.UsersFromBP(customizations.GetUsers())
 	rawImg.Groups = users.GroupsFromBP(customizations.GetGroups())
 
-	rawImg.KernelOptionsAppend = []string{"modprobe.blacklist=vc4"}
+	rawImg.KernelOptionsAppend = []string{"modprobe.blacklist=vc4", "rw"}
```

Comment:
Shouldn't `"ro"` be appended instead? 🤔 I'm probably missing something... Maybe a comment would be nice.